images/win/scripts/Tests/Tools.Tests.ps1

```diff
@@ -4,12 +4,91 @@ Describe "7-Zip" {
     }
 }
 
+Describe "AliyunCli" {
+    It "AliyunCli" {
+        "aliyun version" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "AWS" {
+    It "AWS CLI" {
+        "aws --version" | Should -ReturnZeroExitCode
+    }
+
+    It "Session Manager Plugin for the AWS CLI" {
+        session-manager-plugin | Out-String | Should -Match "plugin was installed successfully"
+    }
+
+    It "AWS SAM CLI" {
+        "sam --version" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "AzCopy" {
+    It "AzCopy" {
+        "azcopy --version" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "Azure Cosmos DB Emulator" {
+    $cosmosDbEmulatorRegKey = Get-ChildItem "HKLM:\SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\*" | Get-ItemProperty | Where-Object { $_.DisplayName -eq 'Azure Cosmos DB Emulator' }
+    $installDir = $cosmosDbEmulatorRegKey.InstallLocation
+
+    It "Azure Cosmos DB Emulator install location registry key exists" -TestCases @{installDir = $installDir} {
+        $installDir | Should -Not -BeNullOrEmpty
+    }
+
+    It "Azure Cosmos DB Emulator exe file exists" -TestCases @{installDir = $installDir} {
+        $exeFilePath = Join-Path $installDir 'CosmosDB.Emulator.exe'
+        $exeFilePath | Should -Exist
+    }
+}
+
+Describe "Azure DevOps CLI" {
+    It "az devops" {
+        "az devops -h" | Should -ReturnZeroExitCode
+    }
+}
+
+Describe "Bazel"{
+    It "<ToolName>" -TestCases @(
+        @{ ToolName = "bazel" }
+        @{ ToolName = "bazelisk" }
+    ) {
+        "$ToolName --version"| Should -ReturnZeroExitCode
+    }
+}
+
 Describe "CMake" {
     It "cmake" {
         "cmake --version" | Should -ReturnZeroExitCode
     }
 }
 
+BeforeAll{
+    $sqlPackagePath = 'C:\Program Files\Microsoft SQL Server\150\DAC\bin\SqlPackage.exe'
+    $sqlLocalDBPath = 'C:\Program Files\Microsoft SQL Server\130\Tools\Binn\SqlLocalDB.exe'
+}
```

Comment:
Could we put this variables directly to `It`? Or move `BeforeAll` block inside `Describe`.
In current implementation this block will be run for any test in this file